lib/tools/system-calls.js

```diff
@@ -75,4 +76,112 @@ systemCallMethods.getBinaryFromPath = async function(binaryName) {
   }
 };
 
+systemCallMethods.getConnectedDevices = async function () {
+  log.debug("Getting connected devices...");
+  try {
+    let {stdout} = await exec(this.adb.path, ['devices']);
```

Comment:
@imurchie this is for convenience. `teen_process.exec()` returns an object that looks like:

```js
{
  stdout: 'some string',
  stderr: 'some string',
  code: 0
}
```

Instead of returning an object, the other option was to return an array. Then we could have retrieved the values like:

```js
let [stdout, stderr, code] = await exec(...);
```

But in many cases we want only one type of thing from `exec`. Let's say we only wanted `stderr`, then it would look like:

```js
let [, stderr] = await exec(...);
```

Which is slightly more mysterious than what we can do now (because of object destructuring):

```js
let {stderr} = await exec(...);
```

In other words, returning an object plus ES6 object destructuring gives us the ability to target simply those return values we're interested in.